packages/core/core/src/BundleGraph.js

```diff
@@ -615,7 +620,7 @@ export default class BundleGraph {
       visitedBundles.add(descendant);
       if (
         descendant.type !== bundle.type ||
-        descendant.env.context !== bundle.env.context
+        !isSubset(descendant.env.context, bundle.env.context)
```

Comment:
`isSubset()` checks `a subset of b OR b subset of a`. But doesn't the previous "context changes" translate to "child context contains something that the parent doesn't support"? So only `child not subset of parent`?